pkgs/development/tools/ammonite/default.nix

```
@@ -1,5 +1,5 @@
-{ stdenv, fetchurl, jre, nixosTests, writeScript, common-updater-scripts, git
-, nixfmt, nix, coreutils, gnused, disableRemoteLogging ? true }:
+{ stdenv, lib, fetchurl, jre, nixosTests, writeScript, common-updater-scripts, git
+, nixfmt, nix, coreutils, gnused, util-linux, disableRemoteLogging ? true }:
 
 with stdenv.lib;
```

```suggestion
with lib;
```

Also I would prefer if lib is moved to the places it is needed.